src/blockrelay/mempool_sync.h

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2018-2019 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_MEMPOOL_SYNC_H
+#define BITCOIN_MEMPOOL_SYNC_H
+
+#include "blockrelay/blockrelay_common.h"
+#include "blockrelay/graphene.h"
+#include "consensus/consensus.h"
+
+#include <chrono>
+
+const uint64_t DEFAULT_MEMPOOL_SYNC_MIN_VERSION_SUPPORTED = 0;
+const uint64_t DEFAULT_MEMPOOL_SYNC_MAX_VERSION_SUPPORTED = 0;
+// arbitrary entropy passed to CGrapheneSet an used for IBLT
+const uint32_t IBLT_ENTROPY = 13;
+// any value greater than 2 will use SipHash
+const uint64_t SHORT_ID_VERSION = 2;
+// frequency of synchronization (per peer) in microseconds
+const int64_t MEMPOOLSYNC_FREQ_US = 30 * 1e6;
+const int64_t MEMPOOLSYNC_FREQ_GRACE_US = 5 * 1e6;
+// Use CVariableFastFilter if true, otherwise use CBloomFilter
+const bool COMPUTE_OPTIMIZED = true;
+
+class CMempoolSyncState
```

Comment:
please add doxygen comments for all classes

Reply:
Done.